### ISSUE TITLE
Facility for padding image files

### ISSUE THREAD

**Issue by rdmark** [akuker/RASCSI#249]
develop 9795c55

**Problem space:**  There are plenty of images (both hard drive and CD-ROM) floating around out there that are of non-standard sizes, i.e. not a multiple of 512 (or 2048 etc.)  Other more lenient pieces of software such as Basilisk II will happily read such files, but RaSCSI won't.

**Solution:**  Just spitballing at this point, but provide some facility to pad such image files, either dynamically at load time, or as a separate tool/UI flow to add the padding permanently.

**Comment by karrots** [comment]
Voting for a separate tool or just CLI option to do the padding. Seems like silently modifying images like that may have unintended consequences if someone assumes the file hasn't changed. Example. Maybe they intended to mount read-only then later a HASH doesn't match and they get confused.

**Comment by uweseimet** [comment]
I agree that rascsi should not do anything like that. rascsi is not meant to be a tool that repairs files etc. In addition, if rascsi is too lenient with using files that are not usable it might cover filesystem issues or other severe issues, which should not be hidden from the user.

**Comment by stinkerton18** [comment]
Attaching an image for a use case. This ISO boots and is installable in BasaliskII and, if burned to a CD, is bootable in drives that can read burned media.

![Screen Shot 2021-09-22 at 10 57 50 PM](https://user-images.githubusercontent.com/12246929/134460651-4aa40dbb-127c-4808-9c32-f6f35bcf172d.png)

**Comment by rdmark** [comment]
@stinkerton18 Can you share the URL where you got the file from?

**Comment by stinkerton18** [comment]
Certainly. I picked up the ISO I'm using from Winworld here: https://winworldpc.com/download/7724c394-e280-9362-c382-11c3a6e28094

**Comment by rdmark** [comment]
@stinkerton18 Thanks for sharing!  Now, I was actually able to attach this image as-is with RaSCSI over here, so maybe the problem you had was with that upload form corruption bug?

Anyhow, I've found two test cases of my own:
https://macintoshgarden.org/apps/macintosh-os-71x (0074 in MacOS71_CD.ZIP)
https://macintoshgarden.org/apps/adam-the-inside-story

Was able to confirm that this method worked for padding:
`dd if=/dev/null of=TheInsideStory.iso bs=1 count=1 seek=456486912`

**Comment by rdmark** [comment]
Implemented in rascsi-web here: https://github.com/akuker/RASCSI/pull/269
